als-server/shared/src/test/scala/org/mulesoft/als/server/workspace/fileusage/FileUsageTest.scala

```diff
@@ -61,6 +60,34 @@ class FileUsageTest extends ServerFileUsageTest {
       ),
       Set(Location("file:///root/api.yaml", Range(Position(5, 13), Position(5, 19))))
     )
+//    TestEntry(
```

Comment:
is it ok for this whole code block to be ignored? it's supposed to remain ignored until an upcoming fix is implemented?

Reply:
It's not yet fixed, depends on a Ticket created in AMF